### PR TITLE
feat(client): add Index price constituents

### PR DESCRIPTION
**Pull request by yzh-pelle**

### CODE REVIEW

**Comment by carlosmiei** [comment]
@yzh-pelle thanks, can you add a test?

**Comment by yzh-pelle** [comment]
> @yzh-pelle thanks, can you add a test?

Sure!

File: binance/async_client.py [hunk 1]
```diff
@@ -1759,6 +1759,9 @@ async def futures_symbol_ticker(self, **params):
     async def futures_orderbook_ticker(self, **params):
         return await self._request_futures_api("get", "ticker/bookTicker", data=params)
 
+    async def futures_index_price_constituents(self, **params):
```
Comment (carlosmiei) [inline binance/async_client.py:1762]: @yzh-pelle we're missing assigning the .Doc here

Comment (yzh-pelle) [reply binance/async_client.py:1762]: Do I need to add to the docstring?
:returns: API response

Or something else?

**Pull request merged**